### ISSUE TITLE
Problems with python engine used

### ISSUE THREAD

**Issue by iN3O** [hugsy/gef#35]
I have installed capstone , keystone and ropgadget using python2.7. 
I have removed python3 binaries from the /usr/bin/ folder but when I open the gdb it still gives me following error.
```
Type "apropos word" to search for commands related to "word".
[*] Failed to load `capstone-disassemble`: 'Missing Python `capstone` package. Install with `pip3 install capstone`'
[*] Failed to load `assemble`: 'Missing Python `keystone` package. Install with `pip3 install keystone`'
[*] Failed to load `ropgadget`: 'Missing Python `ropgadget` package. Install with `pip3 install ropgadget`'
[*] Failed to load `unicorn-emulate`: 'This command requires the following packages: `unicorn` and `capstone`.'
[*] Failed to load `set-permission`: 'Missing Python `keystone` package. Install with `pip3 install keystone`'
gef loaded, `gef help' to start, `gef config' to configure
32 commands loaded (17 sub-commands), using Python engine 3.5
```

How can I use Python2.7 engine ?

**Comment by hugsy** [comment]
On Mon, Jun 20, 2016 at 05:43:48AM -0700, Shubham Bansal wrote:
>
> gef loaded, `gef help' to start, `gef config' to configure
> 32 commands loaded (17 sub-commands), using Python engine 3.5
>
Your gdb is compiled with Python 3.5 bindings.

Either install the dependencies (Keystone, Capstone, Unicorn) for Python 3 or
recompile gdb with python2 bindings.

--
hugsy

"Computer science is no more about computers than astronomy is about telescopes." - E. Dijkstra

**Comment by iN3O** [comment]
Okay. Now I have switched to Python 3.5 and I am doing `pip3 install keystone` but it gives me following error

   ```
Collecting keystone
  Using cached Keystone-0.2.1.tar.gz
    Complete output from command python setup.py egg_info:
    Downloading http://pypi.python.org/packages/source/d/distribute/distribute-0.6.15.tar.gz
    Traceback (most recent call last):
      File "/tmp/pip-build-z21atep1/keystone/distribute_setup.py", line 143, in use_setuptools
        raise ImportError
    ImportError
    
    During handling of the above exception, another exception occurred:
   
 Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-build-z21atep1/keystone/setup.py", line 2, in <module>
        use_setuptools('0.6.15')
      File "/tmp/pip-build-z21atep1/keystone/distribute_setup.py", line 145, in use_setuptools
        return _do_download(version, download_base, to_dir, download_delay)
      File "/tmp/pip-build-z21atep1/keystone/distribute_setup.py", line 124, in _do_download
        to_dir, download_delay)
      File "/tmp/pip-build-z21atep1/keystone/distribute_setup.py", line 196, in download_setuptools
        data = src.read()
      File "/usr/lib/python3.5/http/client.py", line 461, in read
        s = self._safe_read(self.length)
      File "/usr/lib/python3.5/http/client.py", line 607, in _safe_read
        chunk = self.fp.read(min(amt, MAXAMOUNT))
      File "/usr/lib/python3.5/socket.py", line 575, in readinto
        return self._sock.recv_into(b)
      File "/usr/lib/python3.5/ssl.py", line 925, in recv_into
        return self.read(nbytes, buffer)
      File "/usr/lib/python3.5/ssl.py", line 787, in read
        return self._sslobj.read(len, buffer)
      File "/usr/lib/python3.5/ssl.py", line 571, in read
        v = self._sslobj.read(len, buffer)
    ssl.SSLError: [SSL: DECRYPTION_FAILED_OR_BAD_RECORD_MAC] decryption failed or bad record mac (_ssl.c:1977)
    
    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-build-z21atep1/keystone/

```

Also, I tried to install keystone from the git repo but its not detected by the gef.

**Comment by iN3O** [comment]
Found the Fix.

You guys need to update the keystone installation steps. For correct information, check [Keystone-install](https://github.com/keystone-engine/keystone/blob/b806b617386c7d38d46dce77ccd12545af240fb3/docs/COMPILE-NIX.md)

**Comment by hugsy** [comment]
Yes, I was made aware that there is already a (wrong) package called `keystone` in pip. 

Fixed by e6c4f47fc914f7014daa4914e1a9f1bfad8abbe2
